### PR TITLE
Renamed 2 RRect constructors.

### PR DESCRIPTION
**Pull request by dragostis**
RRect.fromLTRBCustom -> RRect.fromLTRBAndCorners
RRect.fromRectCustom -> RRect.fromRectAndCorners

### CODE REVIEW

File: flutter/lib/ui/geometry.dart [hunk 1]
```diff
@@ -818,9 +818,18 @@ class RRect {
 
   /// Construct a rounded rectangle from its left, top, right, and bottom edges,
   /// and topLeft, topRight, bottomRight, and bottomLeft radii.
```
Comment (Hixie) [inline flutter/lib/ui/geometry.dart]: "The corner radii default to right-angled corners (zero radius)" or some such.

File: flutter/lib/ui/geometry.dart [hunk 2]
```diff
@@ -838,8 +847,15 @@ class RRect {
 
   /// Construct a rounded rectangle from its bounding box and and topLeft,
   /// topRight, bottomRight, and bottomLeft radii.
```
Comment (Hixie) [inline flutter/lib/ui/geometry.dart]: ditto

File: flutter/lib/ui/geometry.dart [hunk 3]
```diff
@@ -1242,10 +1258,13 @@ class RRect {
...
-    return 'RRect.fromLTRBCustom(${left.toStringAsFixed(1)}, '
+    return 'RRect.fromLTRBAndCorners(${left.toStringAsFixed(1)}, '
         '${top.toStringAsFixed(1)}, ${right.toStringAsFixed(1)}, '
-        '${bottom.toStringAsFixed(1)}, ${tlRadius.toString()}, '
-        '${trRadius.toString()}, ${brRadius.toString()}, ${blRadius.toString()})';
+        '${bottom.toStringAsFixed(1)}, '
+        'topLeft: ${tlRadius.toString()}, '
+        'topRight: ${trRadius.toString()}, '
+        'bottomRight: ${brRadius.toString()}, '
+        'bottomLeft: ${blRadius.toString()})';
```
Comment (Hixie) [inline flutter/lib/ui/geometry.dart]: no need for the toString()s.

**Comment by dragostis** [comment]
@Hixie Good to go?

**Comment by Hixie** [comment]
LGTM

**Pull request merged**